Formula/vg.rb

```diff
@@ -0,0 +1,49 @@
+class Vg < Formula
+  desc "Virtualgo: Easy and powerful workspace based development for go"
+  homepage "https://github.com/GetStream/vg"
+  url "https://github.com/GetStream/vg/archive/v0.8.0.tar.gz"
+  sha256 "e7267b1306c36b3a4a4d10cb9e7ece66a9436a430bbf15028b82313496c92ebe"
+
+  depends_on "go"
+  depends_on "dep"
+  depends_on "bindfs" => :optional
+
+  def install
+    ENV["GOPATH"] = buildpath
+    vgpath = buildpath/"src/github.com/GetStream/vg"
+    vgpath.install buildpath.children
+    cd vgpath do
+      system "dep", "ensure"
+      system "go", "build", "-o", bin/"vg", "-ldflags", "-w -s -X github.com/GetStream/vg/cmd.Version=#{version}"
```

Comment:
Split this across lines like most of the other examples, ideally. No need for this line to be 100+ chars long.